docs/analyzer/checker_and_analyzer_configuration.md

```diff
@@ -191,3 +191,74 @@ configuration arguments (json in this case) should be in one line :
 ```
 -config="{ "Checks": "clang-diagnostic-*,clang-analyzer-*", "WarningsAsErrors": "", "HeaderFilterRegex": "", "AnalyzeTemporaryDtors": false, "CheckOptions": [ { "key": "google-readability-braces-around-statements.ShortStatementLines", "value": "1" }, { "key": "modernize-loop-convert.MaxCopySize", "value": "16" }, { "key": "modernize-loop-convert.NamingStyle", "value": "CamelCase" }, { "key": "modernize-use-nullptr.NullMacros", "value": "NULL" } ] }"
 ```
+
+# Cppcheck
+
+As of CodeChecker 6.20, Codechecker can now execute the Cppcheck analyzer.
+
+## Configuration
+
+The Cppcheck analyzer can be configured with --analyzer-config cppcheck:* paramterers.
+
+The supporte analyzer configuration items can be listed with `CodeChecker analyzers --analyzer-config cppcheck --details`
+
+As of the CodeChecker 6.20, the following options are supported:
+
+* `cppcheck:addons` A list of cppcheck addon files.
+* `cppcheck:libraries` A list of cppcheck library definiton files.
+* `cppcheck:platform` The platform configuration .xml file.
+
+Please note that for addons and libraries, multiple items can be specified in the following format: --analyzer-config cppcheck:addons <addon.py> --analyzer-config cppcheck:addons <addon2.py>.
+
+Cppcheck only supports a limited number of platforms. Custom bit lengths can be specified with a platform file.
+
+An example platform file from the cppcheck manual:
```

Comment:
```suggestion
An example platform file from the Cppcheck manual:
```
Lower and upper case versions of Cppcheck are mixed throughout in the documentation. This should be common for every occurrence.

Reply:
capitalized them